app/uk/gov/hmrc/apidocumentation/models/Documentation.scala

```diff
@@ -46,6 +54,18 @@ case class APIDefinition(
   lazy val statusSortedActiveVersions = statusSortedVersions.filterNot(v => v.status == APIStatus.RETIRED)
   lazy val defaultVersion = statusSortedActiveVersions.headOption
   lazy val hasActiveVersions = statusSortedActiveVersions.nonEmpty
+  lazy val label: APIDefinitionLabel = if (isTestSupport.getOrElse(false)) {
+    TEST_SUPPORT_API
```

Comment:
I feel like a case statement here would be more idiomatic. It would show we're checking for Some(true) only. Which raises the question, why have an optional boolean instead of a none option which is defaulted to false?

Reply:
I think the `Option[Boolean]` is because play json will error if the param isn't provided:
```scala
import play.api.libs.json._

case class Example(name: String, isTestSupport: Boolean = false)
object Example {
  implicit val format = Json.format[Example]
}

Json.fromJson[Example](Json.parse("""{ "name": "Example" }"""))

// Exiting paste mode, now interpreting.

import play.api.libs.json._
defined class Example
defined object Example
res1: play.api.libs.json.JsResult[Example] = JsError(List((/isTestSupport,List(ValidationError(List(error.path.missing),WrappedArray())))))
```
vs
```scala
import play.api.libs.json._

case class Example(name: String, isTestSupport: Option[Boolean])
object Example {
  implicit val format = Json.format[Example]
}

Json.fromJson[Example](Json.parse("""{ "name": "Example" }"""))

// Exiting paste mode, now interpreting.

import play.api.libs.json._
defined class Example
defined object Example
res2: play.api.libs.json.JsResult[Example] = JsSuccess(Example(Example,None),)
```